code/modules/organs/subtypes/parasite.dm

```diff
@@ -234,11 +247,56 @@
+			SPAN_WARNING("They cut at your mind, removing the growth that infests it. Your head hurts, and you find some things hard to recall - but you remember who you are, and you are free from a truly terrible fate. Perhaps that is some comfort."))
+			to_chat(target, SPAN_CULT("A part of you misses the feeling of it, pulsing inside your skull like a friend, a constant companion. You have survived - but you cannot help but feel empty."))
+			B.max_damage = B.max_damage*0.65 //ok, now you have permanent brain damage
+		if(4)
+			target.visible_message(SPAN_WARNING("As the black k'ois is cut from [target.name], [target.get_pronoun("he")] spasm[target.get_pronoun("end")] wildly, the black k'ois deeply embedded in [target.get_pronoun("his")] brain. You can see the scars it has left - it was not a clean surgery, and [target.get_pronoun("he")] will never fully recover - but [target.get_pronoun("he")] will live, at least."),\
+			SPAN_WARNING("They cut at your mind, removing the growth that infests it. It's hard to remember what you are, without it. Perhaps, if you think hard enough, you can remember your name."))
+			to_chat(target, SPAN_CULT("You have failed Us. We will find another way. Releasing control."))
+			B.max_damage = B.max_damage*0.45 //yeah your brain is severely messed up man
+		if(5)
+			target.visible_message(SPAN_WARNING("As the black k'ois is cut from [target.name], [target.get_pronoun("he")] spasm[target.get_pronoun("end")] wildly, [target.get_pronoun("his")] brain too damaged to function. A horrible choking sound emerges from [target.get_pronoun("his")] mouth, before it is mercifully silenced as [target.get_pronoun("he")] falls still."),\
+			SPAN_DANGER("You feel the cold of the knife cutting and cutting, but it is too late. There is nothing left of you but the Lii'dra's will, now. A final cut - and then you are granted the only escape that still remains to you."))
+			to_chat(target, SPAN_GOOD("Your function is fulfilled. We have no further need of you. Releasing control."))
+			target.death(FALSE) //too late, there is no brain left to save
 	if(all_languages[LANGUAGE_VAURCA] in target.languages && stage >= 3 && !isvaurca(target))
 		target.remove_language(LANGUAGE_VAURCA)
 		to_chat(target, "<span class='warning'>Your mind suddenly grows dark as the unity of the Hive is torn from you.</span>")
+	if(all_languages[LANGUAGE_LIIDRA] in target.languages && stage >= 3)
+		target.remove_language(LANGUAGE_LIIDRA)
 	removed_langs = 0
+	remove_verb(owner, /mob/living/carbon/human/proc/kois_cough)
+	remove_verb(owner, /mob/living/carbon/human/proc/kois_infect)
 	..()
```

Comment:
these messages should respect unconsciousness and the tumor being removed (how is the tumor speaking to you if it's removed? unless i'm misunderstanding.), but you may skip this if it's too much work.

Reply:
The idea is that it's sort of in the process of being removed. I imagine it as sort of a surreal dream-state given that, if it's being surgically removed, it's likely the person is unconscious. Having it only appear when conscious would mean that these messages pretty much never get seen, and given that they're 90% flavor I think it's better this way.

As for respecting the tumor being removed, unless I'm misunderstanding the removed() proc, this should be called when the tumor is getting removed - if they don't have one or it's already been removed, it's not an issue.